packages/plugin-trpc/cli/src/lib/generate.ts

```diff
@@ -0,0 +1,328 @@
+import { Field, Model, Enum } from '@mrleebo/prisma-ast';
```

Comment:
This file only contains functions. We may want to refactor things a bit more and add some unit tests to guarantee everything is working